ethree/build.gradle

```diff
@@ -9,7 +9,7 @@ apply plugin: 'maven'
 apply plugin: 'org.jetbrains.dokka'
 
 group 'com.virgilsecurity'
-version '0.3.0'
+version '0.3.01'
```

Comment:
0.3.1

Reply:
But I was thinking about that .x gives less possible patches. What if we have to make more than 10 patches?